BedrockCommand.cpp

```diff
@@ -112,6 +36,43 @@ void BedrockCommand::_init() {
                 break;
         }
     }
+    _commandCount++;
+}
+
+const string& BedrockCommand::getName() const {
```

Comment:
We probably can, but it's this way because that's what `SINFO` and the related macros use to log the name (which is the whole reason this function exists, really). I would have to update those to use `getPluginName` and see if that breaks anything else.

Reply:
That's actually the only reason this is here at all, so that `getName` works correctly when `SINFO` is used inside a command.

Reply:
can we just stop using getName() inside the SLOGPREFIX macro? i don't get why it's better than just using a string.